python-usfm-parser/src/usfm_grammar/usfm_parser.py

```diff
@@ -360,8 +361,8 @@ def node_2_usx(node, usfm_bytes, parent_xml_node, xml_root_node): # pylint: disa
     elif len(node.children)>0:
         for child in node.children:
             node_2_usx(child, usfm_bytes, parent_xml_node, xml_root_node)
-    else:
-        raise Exception("Encountered unknown element ", str(node))
+    # else:
```

Comment:
This probably a bigger discussion but I think if we are not raising an exception such errors should be logged and reported. This ideally would be done for the whole package. Don't think there is any action needed here though.